alert-policies/amazon-inspector/HighAssessmentRunFindings.yml

```diff
@@ -0,0 +1,27 @@
+name: High Assessment Run Findings
+
+description: |+
+  This alert is triggered if the number of Assessment Run Findings exceeds 100 for 10 minutes.
+
+type: STATIC
+nrql:
+  query: "SELECT sum(`aws.inspector.TotalAssessmentRunFindings`) as 'Query' FROM Metric"
+
+# Function used to aggregate the NRQL query value(s) for comparison to the terms.threshold (Default: SINGLE_VALUE)
+valueFunction: SINGLE_VALUE
+
+# List of Critical and Warning thresholds for the condition
+terms:
+  - priority: CRITICAL
+    # Operator used to compare against the threshold.
+    operator: ABOVE
+    # Value that triggers a violation
+    threshold: 100
+    # Time in seconds; 120 - 3600
+    thresholdDuration: 600
+    # How many data points must be in violation for the duration
+    thresholdOccurrences: ALL
+
```

Comment:
Kindly add a warning condition too.

Reply:
@pkosanam , addressed, I have added the warning condition.